content/docs/command-reference/remove.md

```diff
@@ -9,27 +9,30 @@ optionally delete them).
 usage: dvc remove [-h] [-q | -v] [--outs] targets [targets ...]
 
 positional arguments:
-  targets        stages (found in dvc.yaml) or .dvc files to remove.
+  targets        Tracked files/directories, stage names (found in
+                 dvc.yaml), or .dvc files to remove.
 ```
 
 ## Description
 
-Safely removes `.dvc` files or stages from `dvc.yaml`. This includes deleting
-the corresponding `.gitignore` entries (based on the `outs` fields removed).
+Safely removes tracked files/directories, stage names (found in `dvc.yaml`), or
+`.dvc` files. This includes deleting the corresponding `.gitignore` entries.
 
 > `dvc remove` doesn't remove files from the DVC <abbr>cache</abbr> or
 > [remote storage](/doc/command-reference/remote). Use `dvc gc` for that.
 
```

Comment:
I think it would be great to have an example (maybe just extending the one below) that shows how to remove the data from the cache, although I don't know that current gc functionality is sufficient to really be useful in this scenario.